include/tvm/tir/schedule/schedule.h

```diff
@@ -180,7 +180,8 @@ class ScheduleNode : public runtime::Object {
   virtual void RemoveRV(const ExprRV& expr_rv) = 0;
 
  public:
-  /******** Block/Loop relation ********/
+  /******** Schedule: Sampling ********/
```

Comment:
Is this (and some following) a placeholder for other schedule primitives? It would be better to put a TODO to make it clearer.

Reply:
Yeah we will be upstreaming a handful of primitives in the *near* future, so let's leave it as it is here given it is not "TODO" but "doing" lol